code/src/components/ClearButton.css

```diff
@@ -0,0 +1,23 @@
+.clear-button {
+  margin-bottom: 10px;
+  border-radius: 5px;
+  border: none;
+  height: 70%;
+  width: 50%;
+  color: #fff;
+  background-color: #876957;
+  box-shadow: 2px 3px 3px rgba(0, 0, 0, 0.5);
+
+}
+
+.p-remove-button {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+  font-size: 1em;
+}
```

Comment:
I just realised that you don't need a p inside the button and can place all the text styling inside the button style instead!